test/api/buffer/api_test.go

```diff
@@ -88,6 +94,15 @@ func TestBufferApiE2E(t *testing.T) {
 			}
 		}
 
+		// Start worker node
+		clk := clock.NewMock()
+		clk.Set(time.Time{}.Add(time.Second))
+		opts := []dependencies.MockedOption{dependencies.WithClock(clk), dependencies.WithEtcdNamespace(etcdNamespace), dependencies.WithUniqueID("my-node"), dependencies.WithTestProject(test.TestProject())}
+		workerDeps := bufferDependencies.NewMockedDeps(t, opts...)
+		workerDeps.DebugLogger().ConnectTo(testhelper.VerboseStdout())
+		_, err := service.New(workerDeps)
+		assert.NoError(t, err)
+
```

Comment:
@michaljurecko Neměl bys takhle od pohledu představu proč když nastartuju tady worker, tak mi task vytvořený requestem na vytvoření receiveru v testu (`test/api/buffer/receivers/create-minimal/001-create/request.json`) skončí s chybou `context canceled`? 🙂 
![image](https://user-images.githubusercontent.com/215660/218776289-3e6c40d6-fb23-494a-8add-1534789890ce.png)

Reply:
@JakubMatejka ak to mas naviazane na kontext requestu, tak ked skonci request, tak skonci aj jeho kontext.

Reply:
Teda asi tam budes potrebovat nieco ako:
```
(inside task callback)
ctx, cancel := context.WithTimeout(context.Background(), taskTimeout)
defer cancel()
```

Reply:
Ak som ta spravne pochopil, tomuto nerozumiem "když nastartuju tady worker,"

Reply:
Priklad z workra, kde moze kontext skoncit z ineho dovodu (worker shutdown request):
https://github.com/keboola/keboola-as-code/blob/8341d6baf11c62068f0f6a5befdd575fdea1d964/internal/pkg/service/buffer/worker/service/slice_upload.go#L33-L36

Reply:
Jo, to bylo ono. Díky.